src/components/PDF/ApplicationForm.tsx

```diff
@@ -48,21 +48,22 @@ class ApplicationForm extends Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      formQuestions: undefined,
+      fields: undefined,
       formAnswers: {},
       pdfApplication: undefined,
-      buttonState: '',
+      title: '',
+      description: '',
       submitSuccessful: false,
       currentPage: 0,
       numPages: 0,
-      startDate: new Date(),
       formError: false,
     };
     this.handleChangeFormValueTextField = this.handleChangeFormValueTextField.bind(this);
```

Comment:
remove bind statements and transform them to arrow functions

Reply:
Done